Documentation/gettinghelp.rst

```diff
@@ -4,18 +4,23 @@
 Getting Help
 ############
 
-We use GitHub issues to maintain a list of `Cilium Frequently Asked Questions (FAQ)`_.
-Check there to see if your question(s) is already addressed.
+Cilium is a project  with a growing community, so if you need help there a
+numerous ways to getting help:
```

Comment:
"numerous ways to getting it" ? since it repeated help the other way

Reply:
I would just change this to "There are numerous ways to get help with Cilium if needed"